index.html

```diff
@@ -0,0 +1,54 @@
+<!doctype html>
+
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Portfolio</title>
+        <meta name="description" content="Portfolio">
+        <meta name="author" content="Sanja">
+        <link rel="stylesheet" href="style.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    </head>
+
+    <body>
+        <section id="base">
+            <header>
+                <div id="topHead">
+                    <p id="time">9:41</p>
+                    <ul id="appBar">
+                        <li class="app"><img src="Icons/CellularConnection.png" alt="CellularConnection Icon" width="18" height="11"></li>
+                        <li class="app"><img src="Icons/Wifi.png" alt="WiFi Icon" width="15" height="11"></li>
+                        <li class="app"><img src="Icons/Battery.png" alt="Battery Icon" width="18" height="8"></li>
+                    </ul>
+                    
+                </div>
+                <div id="bottomHead">
+                    <h3 id="logo">Sanja Mandic</h3>
+                    <img class="unionImg" src="Icons/Union.png" alt="manu Icon" width="15" height="11">
+                </div>
+                
+            </header>
+
+            <div id="headline">
+                <h1 id="title">Hey There. I’m Sanja</h1>
+                <h3 id="subtitle">I’m a Software Developer</h3>
+                <p id="mainDescription">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.</p>
+                <ul id="socialMedia">
+                    <li class="imgSoc1"><img src="Icons/Icon-GitHub.png" alt="GitHub Icon" width="18" height="20"></li>
+                    <li class="imgSoc1"><img src="Icons/Icon-Linkedin.png" alt="LinkedIn Icon" width="18" height="20"></li>
+                    <li class="imgSoc1"><img src="Icons/Icon-Angelist.png" alt="Angelist Icon" width="18" height="20"></li>
+                    <li class="imgSoc1"><img src="Icons/Icon-Twitter.png" alt="Twitter Icon" width="18" height="20"></li>
+                    <li class="imgSoc1"><img src="Icons/Icon-Medium.png" alt="Medium Icon" width="18" height="20"></li>
+                </ul>
+                <img id="backImg" src="Icons/illustration-header-mobile.png" alt="illustration-header-mobile" >
```

Comment:
- [ ] [REQUIRED] I kindly suggest that you do not use the `img` tag, but instead use the `background-image` CSS property to add this image through your `style.css` because the background image is for decorative purposes only

Reply:
Thank you for the review, it helps me make my code more profesional. Is it now ready to be merged?

Reply:
Hi @Sanja969,I kindly suggest that you always wait for approval before merging